Zeeguu Reader/Zeeguu Reader/ArticleTableViewCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ArticleTableViewCell.swift
+//  Zeeguu Reader
+//
+//  Created by Jorrit Oosterhof on 27-01-16.
+//  Copyright © 2015 Jorrit Oosterhof.
+// 
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//  
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//  
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+import ZeeguuAPI
+
+class ArticleTableViewCell: UITableViewCell {
+	
+	private var article: Article
+	
+	private var titleField: UILabel
+	private var descriptionField: UILabel
+	private var articleImageView: UIImageView
+	
+	init(article: Article, reuseIdentifier: String?) {
+		self.article = article
+		titleField = UILabel.autoLayoutCapapble()
+		descriptionField = UILabel.autoLayoutCapapble()
+		articleImageView = UIImageView.autoLayoutCapapble()
+		super.init(style: .Default, reuseIdentifier: reuseIdentifier)
+		setupLayout()
+		updateLabels()
+	}
+	
+	required init?(coder aDecoder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
```

Comment:
oki. so that constructor could return nil... but it never does return nil. instead it throws an error. 
why don't you then do something like: 

    	required init?(coder aDecoder: NSCoder) {
        super.init(coder: <#T##NSCoder#>);
	}